app/controllers/api/v1/cell_output_data_controller.rb

```diff
@@ -6,6 +6,8 @@ class CellOutputDataController < ApplicationController
       def show
         cell_output = CellOutput.where(id: params[:id]).take!
 
+        raise Api::V1::Exceptions::CellOutputDataSizeExceedsLimitError if cell_output&.data_size.to_i > CellOutput::MAXIMUM_DOWNLOADABLE_SIZE
```

Comment:
If `cell_output` is null, will `>` raise an error ?

Reply:
will not raise error, because `nil.to_i == 0`

Reply:
Will not call `to_i` if `cell_output` is null

Reply:
![image](https://user-images.githubusercontent.com/5165454/68531086-937a8e80-0349-11ea-94b1-af05ea2b22d0.png)
why?

Reply:
Ohhhh I see, no `&` between `data_size` and `to_i`